src/components/drive-container.tsx

```diff
@@ -6,10 +6,12 @@ import {
     decodeID,
     encodeID,
     getRootPath,
+    toast,
     useFilterPathContent,
     useItemActions,
 } from '@powerhousedao/design-system';
 import path from 'path';
+import { useTranslation } from 'react-i18next';
```

Comment:
are we planning on implementing translation across the app now? because if we are, we'll need to go through and add the translation text to the whole app, which will be a huge job. otherwise we probably should just stick to English for now.

@acaldas what do you think?

Reply:
Don't know what is the expectation to have translation but I guess it doesn't hurt to add it in new code